db/migrate/20170519153350_add_showback_models.rb

```diff
@@ -0,0 +1,73 @@
+class AddShowbackModels < ActiveRecord::Migration[5.0]
+  def up
+    create_table :showback_usage_types do |t|
+      t.string     :category
+      t.string     :description
+      t.string     :measure
+      t.text       :dimensions
+      t.timestamps
+    end
+
+    create_table :showback_events do |t|
+      t.jsonb      :data
+      t.timestamp  :start_time
+      t.timestamp  :end_time
+      t.belongs_to :resource, :type => :bigint, :polymorphic => true, :index => true
+      t.jsonb      :context
+      t.timestamps
+    end
+
+    create_table :showback_price_plans do |t|
+      t.string     :name
+      t.belongs_to :resource, :type => :bigint, :polymorphic => true
+      t.string     :description
+      t.timestamps
+    end
+
+    create_table :showback_rates do |t|
+      t.bigint     :fixed_rate_cents # Columns needed by gem money
```

Comment:
Ugh, I'm surprised it needs a specific column name to include a specific type: cents.  It might get confusing for other currencies that don't have cents.  A more generic term for the amount of the smallest denomination and with a separate column for the denomination type would probably be easier to understand.